src/index.ts

```diff
@@ -169,7 +173,17 @@ export async function init(config: IClientConfig): Promise<IEppoClient> {
 
     // Set the configuration store to the desired persistent store, if provided.
     // Otherwise the factory method will detect the current environment and instantiate the correct store.
-    const configurationStore = configurationStorageFactory(config.persistentStore, false);
+    const configurationStore = configurationStorageFactory(
+      {
+        persistentStore: config.persistentStore,
+        hasChromeStorage: hasChromeStorage(),
+        hasWindowLocalStorage: hasWindowLocalStorage(),
+      },
+      {
+        chromeStorage: hasChromeStorage() && chrome.storage.local,
+        windowLocalStorage: hasWindowLocalStorage() && window.localStorage,
```

Comment:
IMO not great to have to pass these in with the `hasChromeStorage` protection but we don't know if the APIs are available otherwise.